samples/custom-consent-authorizationserver/samples-custom-consent-authorizationserver.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
-	id "org.springframework.boot" version "3.0.0-RC2"
-	id "io.spring.dependency-management" version "1.0.11.RELEASE"
+	id "org.springframework.boot" version "3.0.0"
+	id "io.spring.dependency-management" version "1.1.0"
```

Comment:
Please revert the updates to `io.spring.dependency-management`. Is the `1.1.0` version required for Spring Boot `3.0.0`?

Reply:
Thanks for the review. I reverted to 1.0.11.RELEASE. I upgraded it because there was a newer version available. Both versions work without a problem.